app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -179,7 +176,10 @@ public void run() {
         logger.info(String.format("Analyzing %s", analysisName));
         AppResult result = _analyticJob.getAnalysis();
         result.save();
-        logger.info(String.format("Analysis of %s took %sms", analysisName, System.currentTimeMillis() - analysisStartTimeMillis));
+        long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
+        logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
+        MetricsController.setJobProcessingTime(processingTime);
```

Comment:
Should this be normalized by the size of the job ? Otherwise, the data will be skewed by large jobs. As it stands, the job processing time is directly proportional to the number of tasks. May be divide it by number of tasks ?